frontend/appflowy_flutter/lib/workspace/application/settings/notifications/notification_settings_cubit.dart

```diff
@@ -10,23 +13,31 @@ part 'notification_settings_cubit.freezed.dart';
 
 class NotificationSettingsCubit extends Cubit<NotificationSettingsState> {
   NotificationSettingsCubit() : super(NotificationSettingsState.initial()) {
-    UserSettingsBackendService()
-        .getNotificationSettings()
-        .then((notificationSettings) {
-      _notificationSettings = notificationSettings;
-      emit(
-        state.copyWith(
-          isNotificationsEnabled: _notificationSettings.notificationsEnabled,
-        ),
-      );
-      _initCompleter.complete();
-    });
+    _initialize();
   }
 
   final Completer<void> _initCompleter = Completer();
 
   late final NotificationSettingsPB _notificationSettings;
 
+  Future<void> _initialize() async {
+    final settings =
+        await UserSettingsBackendService().getNotificationSettings();
+    _notificationSettings = settings;
```

Comment:
I guess it was my bad, we can actually just do
```suggestion
    _notificationSettings =
        await UserSettingsBackendService().getNotificationSettings();
```